.changeset/nervous-parents-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
+---
+
+Moved Button to future structure
```

Comment:
To do: We need to fix the existing codemod + make a codemod to revert existing usages of v2 and v3